test/integration/egress_rules.go

```diff
@@ -39,6 +40,14 @@ func (t *egressRules) setup() error {
 
 // TODO: test negatives
 func (t *egressRules) run() error {
+	if !t.Egress {
+		glog.Info("skipping test since egress is missing")
+		return nil
+	}
+	if platform.ServiceRegistry(t.Registry) == platform.EurekaRegistry { // TODO: re-enable once Eureka egress is working
```

Comment:
This should work out of the box in Eureka as well right? These are egress rules that come directly from CRDs. No service registry lookup.